### DISCUSSION TITLE
Why is the horizontal resolution doubled when capturing raw?

### DISCUSSION THREAD

**Discussion by mr-wrinkles** [raspberrypi/picamera2#1044] [Q&A] [answered]
Hi, I am using the HQ camera and I am trying to capture raw cfa data. When I use capture_array() why does the resulting array have the dimensions (3040,8128) when the sensor is 4056x3040?
Sorry if this is a dumb question, I am new to this :)
Thanks!

```
from picamera2 import Picamera2
import time
import numpy as np

picam2 = Picamera2()
config = picam2.create_still_configuration(raw={"format": "SBGGR10"})
picam2.configure(config)
picam2.start()
time.sleep(1)
raw_data = picam2.capture_array("raw")
picam2.stop()
print(raw_data.shape) # Prints (3040, 8128)
```

**Comment by davidplowman** [comment, marked as answer]
Hi, you've asked for unpacked raw pixels so, because they're more than 8-bits wide they have to occupy 2 bytes each. To turn the raw buffer into 16-bit numbers, use `raw_data.view(np.uint16)`.

**Reply by mr-wrinkles** [reply]
Ah I see. Much thanks!
